schema/impl.go

```diff
@@ -41,19 +38,23 @@ func newRealCluster() *realCluster {
 	return cluster
 }
 
+// GetAllClusterData returns a pointer to the ClusterInfo, along with all of its metrics.
+// GetAllClusterData also returns the latest cluster timestamp, for reuse in GetNew* methods.
 func (rc *realCluster) GetAllClusterData() (*ClusterInfo, time.Time, error) {
```

Comment:
Interface-level methods will be redesigned alongside the REST endpoints.
The existing Getters serve mostly as proof-of-concept endpoints. After the redesign, these methods will probably be renamed to internal methods used only for testing.
Estimate: [Schema 6]

Fundamental ideas:
- Different methods for historical and derived data
- (start, end) parameters for historical data
- Getters return (time, error) and accept a callback function which performs the appropriate conversion and copy of the internal types to external types.